src/Repository/CheckListRepository.php

```diff
@@ -5,54 +5,51 @@
 namespace App\Repository;
 
 use App\Entity\Checkbox;
-use App\Entity\Project;
-use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
-use Doctrine\Persistence\ManagerRegistry;
-
-/**
- * @method Checkbox|null find($id, $lockMode = null, $lockVersion = null)
- * @method Checkbox|null findOneBy(array $criteria, array $orderBy = null)
- * @method Checkbox[] findAll()
- * @method Checkbox[] findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
- *
- * @template T
- * @extends ServiceEntityRepository<T>
- */
-final class CheckListRepository extends ServiceEntityRepository
+use App\Exception\ShouldNotHappenException;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+
+final class CheckListRepository
```

Comment:
I got bit confused here. The repository name != entity name.
Checklist or Checkbox?


- Project has many Checklists
- Checklist has many Checkbox